modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/frontend/data/set/provider/ReplacementCPInstanceFDSDataProvider.java

```diff
@@ -68,7 +67,7 @@ public List<ReplacementSku> getItems(
 		long cProductId = ParamUtil.getLong(httpServletRequest, "cProductId");
 
 		if (sort == null) {
-			sort = SortFactoryUtil.getSort(CPInstance.class, "sku", "ASC");
+			sort = new Sort("sku", false);
```

Comment:
`new Sort("sku", Sort.STRING_TYPE, false)` to be more aligned with the other usages of this

Reply:
@smottal thanks, totally forgot to check how it's used in other places. Fixed it.